src/rustllvm/RustWrapper.cpp

```diff
@@ -985,24 +985,35 @@ extern "C" int64_t LLVMRustDIBuilderCreateOpPlusUconst() {
   return dwarf::DW_OP_plus_uconst;
 }
 
+// Adds YK labels as close to the beginning of BasicBlocks as possible.
```

Comment:
Should this be a `///` doc comment?

Reply:
Note it's C++.

Reply:
Yes, but you can still add doc comments to it.

Reply:
I'm happy to be proven wrong but I don't think `///` means anything to C++ and rustdoc doesn't look at C++ files.

The closesnt markup to `///` would be for doxygen:
```
/**
 * blah...
 */
```

But we don't use doxygen.

Reply:
I'm an idiot -- ignore me!